src/util/chains.ts

```diff
@@ -1,3 +1,5 @@
+import { Currency, Ether, NativeCurrency, Token } from '@uniswap/sdk-core';
+import { WRAPPED_NATIVE_CURRENCY } from '..';
```

Comment:
nit: wonder if it would just make sense to be more explicit with where this dependency is coming from for consistency and readability.